data/items/items.xml

```diff
@@ -37233,7 +37267,6 @@
 		<attribute key="attack" value="20"/>
 		<attribute key="defense" value="30"/>
 		<attribute key="weight" value="2900"/>
-		<attribute key="upgradeclassification" value="2"/>
```

Comment:
Certain?

Reply:
Yes, just confirmed on tibia global, and on tibiawiki sites, none of the ferumbras staff's on the game have upgrade classification, and with this tag the client is crashing